src/os_crypto/shared/keys.c

```diff
@@ -673,6 +673,23 @@ int OS_DeleteSocket(keystore * keys, int sock) {
 int w_get_agent_net_protocol_from_keystore(keystore * keys, const char * agent_id) {
 
     const int key_id = OS_IsAllowedID(keys, agent_id);
-    
+
     return (key_id >= 0 ? keys->keyentries[key_id]->net_protocol : key_id);
 }
+
+int w_get_key_hash(keyentry *key_entry, os_sha1 output) {
```

Comment:
Please, apply an indentation for the function according to the indentation used in the rest of the file.

Reply:
You are right, fixed !